src/vse_sync_pp/parsers/parser.py

```diff
@@ -64,6 +68,7 @@ def relative_timestamp(parsed, tzero):
         parsed = parsed._replace(timestamp=timestamp - tzero)
     return tzero, parsed
 
+
 class Parser():
     """A base class providing common parser functionality"""
     def make_parsed(self, elems): # pylint: disable=no-self-use
```

Comment:
If we are going with flake8 I would take out all lines we previously added  based on adding `# pylint` ?

Reply:
Removed